lithium/lithium.py

```diff
@@ -55,7 +55,8 @@ def readTestcase(self, filename):
             # Determine whether the f has a DDBEGIN..DDEND section.
             for line in f:
                 if line.find(b"DDEND") != -1:
-                    raise LithiumError("The testcase (%s) has a line containing 'DDEND' without a line containing 'DDBEGIN' before it." % self.filename)
+                    raise LithiumError("The testcase ({}) has a line containing 'DDEND' ".format(self.filename) +
+                                       "without a line containing 'DDBEGIN' before it.")
```

Comment:
I liked switching to `.format()` because I don't have to think about the return type of whatever I would print. i.e. Is it `%d`, or `%s`? Or maybe `%f`?

However, I agree about not intermixing styles, and perhaps the switch to `.format()` should be in another PR where needed. Thoughts?